source/ftrack_connect_pipeline_qt/ui/asset_manager/asset_manager.py

```diff
@@ -696,21 +726,41 @@ def set_asset_info(self, asset_info):
         self._component_and_version_header_widget.set_version(
             asset_info[asset_constants.VERSION_NUMBER]
         )
-        self._is_latest_version = asset_info[asset_constants.IS_LATEST_VERSION]
+        self._is_latest_version = version_entity['is_latest_version']
         self._component_and_version_header_widget.set_latest_version(
             self._is_latest_version
         )
+        indicator_color = 'gray'
+        self._is_loaded = asset_info.get(asset_constants.OBJECTS_LOADED) in [
+            True,
+            'True',
```

Comment:
Why checking booth? should only be a boolean right?

Reply:
Maybe it is an artefact from from earlier iterations, will remove it.